LibrariesPrototype/Presentation/Extensions/UIImageView+AsyncLoadingFromURL.swift

```diff
@@ -10,9 +10,7 @@ import UIKit
 let imageCache = NSCache<NSString, UIImage>()
```

Comment:
you already changed it to the alamofireImage but, I really wonder that who is responsible for this object.  I will ask @Orcun